### ISSUE TITLE
[BUG] Error messages upon creating account

### ISSUE THREAD

**Issue by compcon** [massif-press/compcon#1891]
<strong>Submitted At: </strong> 4/3/2022 0:05:38<br><br><strong>Browser: </strong> Firefox<br><strong>Platform: </strong> Desktop<br><br><strong>Issue Description: </strong><br> I created an account on the COMP/CON website, and immediately upon verifying my account and logging in, I got got three instances of the error message "this.userProfile is undefined" and two instances of the error message "this.user is undefined".<br><br><strong>Reproduction Steps:<br> </strong> 1.) Create new COMP/CON account on the website, using the latest desktop version of Firefox for Windows 10. 2.) Enter the account verification code that was sent to your email address into the text field asking for it on the COMP/CON website. 3.) Log into the new account. A settings window for syncing content to the cloud should come up, and along with it, at least one error message as described in the bug description. Saving the sync settings (without changing any from their default state, just clicking the "save" button) causes more error messages to pop up as well.<br><br><strong>Expected Result:<br> </strong> No error messages upon account creation.<br><br><strong>Additional Context:<br> </strong> I don't know if this is related to this same problem or not, but after creating my account, I imported two .lcp files--the free, player-facing files for The Long Rim and No Room for a Wallflower, downloaded directly from the linked itch.io pages--and got the warning that they were out of date, when in fact their version numbers were both one higher than the version number listed by COMP/CON as the latest version. Either I'm somehow accessing an out-of-date version of the website, or the website doesn't know the most recent version of those two files. If the former is the case, that could be what's causing the error messages described in this bug report, which is why I'm including it here.<br><br><br><strong>Stack Trace: </strong> I got five error messages and so I'm pasting in the five stack traces in chronological order.

```
Vue error at [e]
isManualOnly@https://compcon.app/app.bundle.js:343:775679
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.evaluate@https://compcon.app/app.bundle.js:22:27760
bn/<@https://compcon.app/app.bundle.js:22:29669
et@https://compcon.app/app.bundle.js:343:767369
t.prototype._render@https://compcon.app/app.bundle.js:22:35296
i@https://compcon.app/app.bundle.js:22:64441
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.run@https://compcon.app/app.bundle.js:22:27494
pn@https://compcon.app/app.bundle.js:22:25602
ee/<@https://compcon.app/app.bundle.js:22:12293
Jt@https://compcon.app/app.bundle.js:22:11737
```
```
Vue error at [e]
isManualOnly@https://compcon.app/app.bundle.js:343:775679
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.evaluate@https://compcon.app/app.bundle.js:22:27760
bn/<@https://compcon.app/app.bundle.js:22:29669
et@https://compcon.app/app.bundle.js:343:767369
t.prototype._render@https://compcon.app/app.bundle.js:22:35296
i@https://compcon.app/app.bundle.js:22:64441
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.run@https://compcon.app/app.bundle.js:22:27494
pn@https://compcon.app/app.bundle.js:22:25602
ee/<@https://compcon.app/app.bundle.js:22:12293
Jt@https://compcon.app/app.bundle.js:22:11737
```
```
Vue error at [e]
isManualOnly@https://compcon.app/app.bundle.js:343:775679
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.evaluate@https://compcon.app/app.bundle.js:22:27760
bn/<@https://compcon.app/app.bundle.js:22:29669
et@https://compcon.app/app.bundle.js:343:767369
t.prototype._render@https://compcon.app/app.bundle.js:22:35296
i@https://compcon.app/app.bundle.js:22:64441
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.run@https://compcon.app/app.bundle.js:22:27494
pn@https://compcon.app/app.bundle.js:22:25602
ee/<@https://compcon.app/app.bundle.js:22:12293
Jt@https://compcon.app/app.bundle.js:22:11737
```
```
Vue error at [e]
userTheme@https://compcon.app/app.bundle.js:198:106252
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.evaluate@https://compcon.app/app.bundle.js:22:27760
bn/<@https://compcon.app/app.bundle.js:22:29669
created@https://compcon.app/app.bundle.js:198:106309
Ut@https://compcon.app/app.bundle.js:22:11265
Qe@https://compcon.app/app.bundle.js:22:25181
t.prototype._init@https://compcon.app/app.bundle.js:22:32825
a@https://compcon.app/app.bundle.js:22:30399
init/t.componentInstance<@https://compcon.app/app.bundle.js:22:19298
init@https://compcon.app/app.bundle.js:22:19329
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
S/<@https://compcon.app/app.bundle.js:22:55650
S@https://compcon.app/app.bundle.js:22:55834
lo@https://compcon.app/app.bundle.js:22:56970
t.prototype._update@https://compcon.app/app.bundle.js:22:34334
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.run@https://compcon.app/app.bundle.js:22:27494
pn@https://compcon.app/app.bundle.js:22:25602
ee/<@https://compcon.app/app.bundle.js:22:12293
Jt@https://compcon.app/app.bundle.js:22:11737
```
```
Vue error at [e]
userID@https://compcon.app/app.bundle.js:198:106210
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.evaluate@https://compcon.app/app.bundle.js:22:27760
bn/<@https://compcon.app/app.bundle.js:22:29669
r@https://compcon.app/app.bundle.js:198:99798
t.prototype._render@https://compcon.app/app.bundle.js:22:35296
i@https://compcon.app/app.bundle.js:22:64441
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
h@https://compcon.app/app.bundle.js:22:53378
d@https://compcon.app/app.bundle.js:22:52900
lo@https://compcon.app/app.bundle.js:22:56922
t.prototype._update@https://compcon.app/app.bundle.js:22:34351
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un@https://compcon.app/app.bundle.js:22:26528
kn.prototype.$mount/<@https://compcon.app/app.bundle.js:22:64455
kn.prototype.$mount@https://compcon.app/app.bundle.js:22:64602
init@https://compcon.app/app.bundle.js:22:19337
d/<@https://compcon.app/app.bundle.js:22:52509
d@https://compcon.app/app.bundle.js:22:52786
S/<@https://compcon.app/app.bundle.js:22:55650
S@https://compcon.app/app.bundle.js:22:55834
lo@https://compcon.app/app.bundle.js:22:56970
t.prototype._update@https://compcon.app/app.bundle.js:22:34334
i@https://compcon.app/app.bundle.js:22:64431
un.prototype.get@https://compcon.app/app.bundle.js:22:26610
un.prototype.run@https://compcon.app/app.bundle.js:22:27494
pn@https://compcon.app/app.bundle.js:22:25602
ee/<@https://compcon.app/app.bundle.js:22:12293
Jt@https://compcon.app/app.bundle.js:22:11737
```<br>

**Comment by msprijatelj** [comment]
I believe that CompCon doesn’t handle the initial account creation super well (as evidenced here). Refreshing the page usually solves most issues, but it would be better for user experience if all the pertinent data updated without requiring a refresh. 

As for the LCP version numbers: The version numbers are pulled from a file hosted in S3. It is possible that the version file was updated in the meantime. There is already an existing issue for LCP version numbers more recent than the ones on file, though we do have some methods in place for homebrew authors to submit their updated versions (the Massif LCPs are on us).

**Comment by jarena3** [comment]
Fixed in 2.3
